src/classes/OPP_OpportunityNaming_BATCH_TEST.cls

```diff
@@ -67,152 +66,46 @@ private with sharing class OPP_OpportunityNaming_BATCH_TEST {
-        }
-    }
-
-    /**
-     * @description Confirms that batch query locator is limited
-     */
-    @isTest
-    private static void shouldLimitQuery() {
-        OPP_OpportunityNaming_BATCH batch = new OPP_OpportunityNaming_BATCH();
-        String query = batch.start(null).getQuery();
-        System.assert(query.endsWith('LIMIT 10000000'), 'The query should have the correct limit');
-    }
-
-    /**
-     * @description Confirms that the execute method tracks the Id of the last Opportunity processed
-     */
-    @isTest
-    private static void shouldTrackLastOpportunityIdProcessed() {
-        List<Opportunity> testOpps = [SELECT Id, Name FROM Opportunity ORDER BY Id];
-
-        OPP_OpportunityNaming_BATCH batch = new OPP_OpportunityNaming_BATCH();
-
-        Test.startTest();
-        batch.execute(null,testOpps);
-        Test.stopTest();
-
-        Id expectedId = testOpps[NUM_OPPS - 1].Id;
-        System.assertEquals(expectedId, batch.lastOppIdProcessed,
-            'The execute method should track the Id of the last opportunity processed');
-    }
-
-    /**
-     * @description Confirms that the finish method chains the next batch with an offset
-     * of the last Opportunity Id processed
+     * @description Constructs Batch with a different Batch Size
      */
     @isTest
-    private static void shouldChainNextBatchOffsetByLastRecordProcessed() {
-        List<Opportunity> testOpps = [SELECT Id FROM Opportunity ORDER BY Id];
-
-        OPP_OpportunityNaming_BATCH batch = new OPP_OpportunityNaming_BATCH();
-        batch.lastOppIdProcessed = testOpps[0].Id;
-
-        Test.startTest();
-        batch.finish(null);
-        Test.stopTest();
-
-        List<AsyncApexJob> jobsApexBatch = queryOppNamingBatchJobs();
-        System.assertEquals(1, jobsApexBatch.size(), 'The naming batch should be started again');
-
-        Opportunity offsetOpportunity = [SELECT Name FROM Opportunity WHERE Id = :testOpps[0].Id];
-        System.assert(offsetOpportunity.Name.startsWith('Test Opp'),
-            'The offset opportunity should not have been processed in the chained batch');
-
-        for (Opportunity opp : [SELECT Name FROM Opportunity WHERE Id != :testOpps[0].Id]) {
-            System.assertEquals(false, opp.Name.startsWith('Test Opp'),
-                'The remaining opportunities should have been renamed: ' + opp.Name);
-        }
+    private static void constructBatchWithDifferentBatchSize() {
+        OPP_OpportunityNaming_BATCH batch = new OPP_OpportunityNaming_BATCH(10);
+        System.assertEquals (10, batch.getBatchSize(), 'Batch Size');
     }
 
     /**
-     * @description Confirms that the finish method does not chain the next batch
-     * if it fails to capture the last Opportunity Id processed
+     * @description Coverage of Schedulable Method
      */
-    @isTest
-    private static void shouldNotChainNextBatchIfLastOppIdProcessedIsNull() {
+    static testMethod void testSchedulable() {
```

Comment:
This should use @IsTest to remain consistent with all other methods in this class